system-parachains/common/src/randomness.rs

```diff
@@ -81,17 +86,29 @@ where
 			.inspect_err(|e| {
 				log::error!(
 					target: LOG_TARGET,
-					"Failed to lookup `well_known_keys::ONE_EPOCH_AGO_RANDOMNESS` from trie: {e}"
+					"Failed to lookup `well_known_keys::ONE_EPOCH_AGO_RANDOMNESS` from the \
+					validation data. The node may have maliciously omitted it. Error: {e}"
```

Comment:
we could also panic here, to enforce the collator to construct correctly the validation data.

Reply:
I did not know that the collator can just omit RC proof data?! Hm, sounds like a valid reason to panic, yes.  
WDYT @skunert?